### PR TITLE
Capitalize error code

### PR DESCRIPTION
**Pull request by Rageking8**
The "C" in error codes should be capitalized.

### CODE REVIEW

File: docs/error-messages/compiler-warnings/compiler-warning-level-4-c4389.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
+description: "Learn more about: Compiler Warning (level 4) C4389"
 title: "Compiler Warning (level 4) C4389"
-description: "Microsoft C/C++ compiler warning C4389, its causes and resolution."
```
Comment (Rageking8) [inline docs/error-messages/compiler-warnings/compiler-warning-level-4-c4389.md:3]: Normalized this description.

**Comment by ShannonLeavitt** [comment]
@TylerMSFT

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by TylerMSFT — APPROVED** [review]
All good - thank you

**Comment by TylerMSFT** [comment]
#sign-off

File: docs/error-messages/compiler-warnings/compiler-warning-level-4-c4389.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
+description: "Learn more about: Compiler Warning (level 4) C4389"
 title: "Compiler Warning (level 4) C4389"
```
Comment (Jak-MS) [inline docs/error-messages/compiler-warnings/compiler-warning-level-4-c4389.md:3]:
```suggestion
title: "Compiler Warning (level 4) C4389"
description: "Learn more about: Compiler Warning (level 4) C4389"
```

**Pull request merged**